examples/fluid/sphere_surface_tension_2d.jl

```diff
@@ -46,7 +46,10 @@ tspan = (0.0, 5.0)
 ode = semidiscretize(semi, tspan)
 
 info_callback = InfoCallback(interval=100)
-saving_callback = SolutionSavingCallback(dt=0.02)
+
+# For overwriting via trixi_include
+saving_interval = 0.02
+saving_callback = SolutionSavingCallback(dt=saving_interval)
```

Comment:
Why? One can just set `dt=0.01`. The `dt` below for the initial time step will be overwritten too, but it's irrelevant.

Reply:
yes but this is in no way logical that dt changes the output.

Reply:
Well, none of this is documented anywhere, so one has to look into the example file anyway. And it's `dt` for all other example files.